Makefile

```diff
@@ -125,3 +128,14 @@ android-ports: ##@other Add reverse proxy to Android Device/Simulator
 	adb reverse tcp:8081 tcp:8081
 	adb reverse tcp:3449 tcp:3449
 	adb reverse tcp:4567 tcp:4567
+
+start-dev:
+	${MAKE} prepare
+ifeq ($(SYSTEM),android)
+	${MAKE} android-ports
+	${MAKE} dev-android-$(DEVICE)
+else
+	${MAKE} prepare-ios
+	${MAKE} dev-ios-real
+endif
+	${MAKE} -j3 react-native repl-$(SYSTEM) run-$(SYSTEM)
```

Comment:
what is `repl-$(SYSTEM)` ?

Reply:
would run repl-android or repl-ios, based on the variabe.

Basically it would run fighweel + refrisk and give you a repl, and will keep compiling the files.